sdk/websocket.go

```diff
@@ -19,28 +19,75 @@ type serviceWs struct {
 	client *ClientWs
 }
 
-var chanSubscribe = struct {
-	block  chan BlockInfo
-	status chan StatusInfo
-}{}
-
 // Catapult Websocket Client configuration
 type ClientWs struct {
```

Comment:
Can we rename this to ClientWebsocket instead?

Reply:
Insurance!